src/gsi/read_satwnd.f90

```diff
@@ -868,6 +874,33 @@ subroutine read_satwnd(nread,ndata,nodata,infile,obstype,lunout,gstime,twind,sis
                     if(itype == 240 .and. ppb < 700.0_r_kind) qm=15
                  endif
               endif
+           else if( trim(subset) == 'NC005072') then ! LEOGEO (LeoGeo)  winds
+              if(hdrdat(1) ==854 ) then              ! LEOGEO satellite ID
+                 c_prvstg='LEOGEO'
+                 if(hdrdat(9) == one)  then          !LEOGEO IR winds
+                    itype=255
+                    c_station_id='IR'//stationid
+                    c_sprvstg='IR'
+                 endif
+! get quality information
+                 call ufbrep(lunin,qcdat,3,8,iret,qcstr)
+                 !!! Rethink this strategy!!!
+                 qifn=qcdat(3,1)
+                 qify=qcdat(3,2)
+                 ee  =qcdat(3,3)
+                 !do j=1,6
+                    !if( qify <=r105 .and. qifn <r105 .and. ee <r105) exit
+                    !if(qcdat(2,j) <= r10000 .and. qcdat(3,j) <r10000 ) then
+                    !   if(qcdat(2,j) ==  one  .and. qifn >r105) then
+                    !      qifn=qcdat(3,j)
+                    !   else if(qcdat(2,j) ==  three .and. qify >r105) then
+                    !      qify=qcdat(3,j)
+                    !   else if( qcdat(2,j) == four .and. ee >r105) then
+                    !      ee=qcdat(3,j)
+                    !   endif
+                    !endif
+                    !enddo
```

Comment:
If these commented out lines aren't needed and won't be used in the future, please remove them.